eth/protocols/eth/handshake.go

```diff
@@ -51,6 +51,8 @@ func (p *Peer) Handshake(network uint64, td *big.Int, head common.Hash, genesis
 			Head:            head,
 			Genesis:         genesis,
 			ForkID:          forkID,
+			// Step 2
```

Comment:
should consider multi-client compatibility

Reply:
for erigon and reth,
only can parse the optional JustifiedNumber is ok